base/strings/search.jl

```diff
@@ -127,16 +127,17 @@ findfirst(ch::AbstractChar, string::AbstractString) = findfirst(==(ch), string)
 function findnext(testf::Function, s::AbstractString, i::Integer)
```

Comment:
What's going on with these changes? Seems like they don't belong in here.

Reply:
yeah, but how do I remove it? I don't know how to rm this commit, either don't how did I got it into this PR...

Reply:
You'll have to undo https://github.com/JuliaLang/julia/pull/31069/commits/a47cf0084575a8beba0d2eec1701cfc427af2056

[Undo a commit](https://ohshitgit.com/#undo-a-commit)


```
git revert a47cf0084575a8beba0d2eec1701cfc427af2056
```